addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1546,7 +1555,7 @@ msgid "Screensaver"
 msgstr ""
 
 msgctxt "#361"
-msgid "R. Slideshow"
+msgid "R. slideshow"
```

Comment:
Ah, thx, yes  this been nagging me, I wonder if label wont scroll, I hate scrolling labels, but yes, Ill change that.